group_vars/figgy/staging.yml

```diff
@@ -31,6 +31,7 @@ figgy_rabbit_password: '{{vault_rabbit_staging_password}}'
 figgy_rabbit_server: 'amqp://{{figgy_rabbit_user}}:{{figgy_rabbit_password}}@{{figgy_staging_rabbit_host}}:5672'
 figgy_geoserver_user: '{{ vault_figgy_geoserver_user }}'
 figgy_geoserver_password: '{{ vault_figgy_geoserver_password }}'
+figgy_staging_rabbit_host: 'figgy-web-staging1.princeton.edu'
```

Comment:
I did a search for where this var is getting used, and found [this reference](https://github.com/pulibrary/princeton_ansible/blob/c368eed1c2c486b02e07189f024c7ed0b4a21292/group_vars/figgy/staging.yml#L31) in the figgy staging group vars. Does port 5672 need to be open on figgy-web-staging1?

Reply:
Yeah probably.